components/ISCNForm.vue

```diff
@@ -67,7 +80,12 @@
 
     <!-- Author Info -->
     <div class="grid grid-cols-2 gap-4">
-      <UFormGroup label="Author Name" required>
+      <UFormGroup
+        label="Author Name"
+        :error="(!formData.author.name) && 'Author name is required'"
```

Comment:
Unnecessary brackets

```suggestion
        :error="!formData.author.name && 'Author name is required'"
```

Reply:
[🎨 Improve condition checks for required fields in ISCNForm](https://github.com/likecoin/nft-book-press/pull/352/commits/9f7b1286287b8c05cb457e002f367f1161c8eef3)